tasks/section_4/cis_4.6.x.yml

```
@@ -89,17 +89,22 @@
 
 - name: "4.6.5 | PATCH | Ensure default user umask is 027 or more restrictive"
   block:
-      - name: "4.6.5 | PATCH | Ensure default user umask is 027 or more restrictive | Set umask for /etc/login.defs pam_umask settings"
+      - name: "4.6.5 | PATCH | Ensure default user umask is 027 or more restrictive | Set umask for /etc/login.defs and /etc/profile"
         ansible.builtin.lineinfile:
             path: "{{ item.path }}"
-            regexp: '(?i)(umask\s*)'
+            regexp: '(?i)(umask\s*\d\d\d)'
```

umask can be 0027 or 027
should check for lines without comments
no need for capture group

```suggestion
            regexp: '(?i)^\s*umask\s*\d{3,4}'
```

---

Agreed this should be able to capture both types 3 & 4

---

And the `^\s*` ensure the line starts with `umask`, and the line is not commented out

---

I see, I will add these modifications!